web/src/main/java/no/nav/foreldrepenger/tilbakekreving/web/app/tjenester/los/LosBehandlingDtoTjeneste.java

```diff
@@ -65,6 +65,8 @@ public LosBehandlingDto lagLosBehandlingDto(Behandling behandling) {
             false,
             null,
             null,
+            null,
+            List.of(),
```

Comment:
gjerne dette:
   false,
   List.of(),
   null,
   null,
   List.of(),
   mapTilbake(behandling, kravgrunnlag431, frist));